analysis/parser/RawTextParser/src/test/kotlin/de/maibornwolff/codecharta/rawtextparser/MetricCollectorTest.kt

```diff
@@ -1,14 +1,24 @@
 package de.maibornwolff.codecharta.rawtextparser
 
 import de.maibornwolff.codecharta.parser.rawtextparser.MetricCollector
+import de.maibornwolff.codecharta.parser.rawtextparser.RawTextParser
 import org.assertj.core.api.Assertions
 import org.junit.jupiter.api.Test
 import java.io.File
 
 class MetricCollectorTest {
+
+    private val defaultVerbose = false
+    private val defaultMaxIndentLvl = RawTextParser.DEFAULT_INDENT_LVL
```

Comment:
I don't believe it's necessary to reference the default value of the RawTextParser for the defaultMaxIndentLvl, as we don't refer to its defaultVerbose or similar attributes. It would also leads to less coupled tests.